packages/react-router/src/fileRoute.ts

```diff
@@ -6,23 +6,17 @@ import { useLoaderData } from './useLoaderData'
 import { useSearch } from './useSearch'
 import { useParams } from './useParams'
 import { useNavigate } from './useNavigate'
-import type { NoInfer } from '@tanstack/react-store'
+import { type Constrain } from './utils'
```

Comment:
```suggestion
import type { Constrain } from './utils'
```
Just for consistency.